addons/rook-ceph/enable

```diff
@@ -0,0 +1,106 @@
+#!/usr/bin/env python3
+
+import os
+import pathlib
+import subprocess
+
+import click
+
+SNAP = pathlib.Path(os.getenv("SNAP") or "/snap/microk8s/current")
```

Comment:
Same here, when is the or section called?

Reply:
Similar to above. 
Also, same style for declaring `SNAP` var in other addons, like GPU, kube-ovn etc.

Reply:
Under which condition the "or" section is run? I suspect this script should not be called outside the context of the SNAP.

Reply:
Done